docs/architecture/adr-081-protobuf-mgmt.md

```diff
@@ -97,7 +99,8 @@ members. Maintainability is a core concern here.
 One of the primary considerations regarding the usage of Buf is whether, for
 example, access to its registry will eventually become a
 paid-for/subscription-based service and whether this is valuable enough for us
-and the ecosystem to pay for such a service.
+and the ecosystem to pay for such a service. At this time, it appears as though
+Buf will never charge for hosting open source projects' protos.
```

Comment:
For the record, I'm less concerned about whether or not we have to pay, versus whether the service will still be there in a couple years time. Cost matters, but at this point I think the larger risk is availability.